STM32Cube/Tasks/log.c

```diff
@@ -179,7 +189,7 @@ void logTask(void *argument) {
 
     // wait for a full buffer to appear in the queue; timeout is long - queues are not expected to fill up super quickly
     for (;;) {
-		if (xQueueReceive(fullBuffersQueue, &bufferToPrint, 1000000) == pdPASS) {
+		if (xQueueReceive(fullBuffersQueue, &bufferToPrint, 10000) == pdPASS) {
```

Comment:
This doesn't actually change how fast we log buffers since all the task does is block on this and repeat. Christine added some more testing and monitoring into the loop though so I think this is for that